blocks/blocks.go

```diff
@@ -19,48 +20,61 @@ type Block interface {
 	Loggable() map[string]interface{}
 }
 
-// Block is a singular block of data in ipfs
-type RawBlock struct {
+type BasicBlock struct {
```

Comment:
aww, renaming this from `RawBlock` to `BasicBlock` makes the diff so noisy... I was hoping to avoid that by getting that change merged separately